TCPEchoServer/Server.cpp

```diff
@@ -0,0 +1,81 @@
+﻿#define WIN32_LEAN_AND_MEAN
+
+#include <iostream>
+#include "Socket.h"
+int main(int argc, char* argv[])
+{
+
+	WSADATA wsaData;
+
+	int iResult = WSAStartup(MAKEWORD(2, 2), &wsaData);
+	if (iResult != 0)
+	{
+		std::cout << "WSAStartup failed: " << iResult << std::endl;
+	}
+
+	{
+		Socket listenSocket{};
+		if (argc == 0)
+		{
+			std::cout << "No command line arguments, please, define " << std::endl;
+			return 1;
+		}
+
+		if (argc == 1)
+		{
+			std::cout << "Only one command line argument, please, define the second " << std::endl;
+			return 1;
+		}
+
+		unsigned long long port;
+		try
+		{
+			iResult = listenSocket.bind(argv[1], port = std::stoull(argv[2]));
```

Comment:
Split this. Leave the port assignment in the try/catch block. Other will place down the code.

Reply:
Done